complex_import/manual.md

```diff
@@ -58,32 +58,50 @@ To make it easier to match objects after the import, the naming convention for a
 
 * Cylinder/TX+2   (a cylinder object which is translated along the X axis by 2 units in the positive direction)
 * Cone/TY+5/RX-90  (a cone which is translated along Y by +5 units, and rotated around X by -90 degrees)
+* Meshes have a 'Mesh' suffix in the name, Materials have a 'Mat_' prefix
```

Comment:
Is it also 'Mesh_' with underscore like 'Mat_' or 'Mesh'?

Reply:
Nope, it's just as it says. Not sure if worth the effort to change... It's a lot of work